pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/MinAggregationFunction.java

```diff
@@ -40,6 +40,11 @@ public String getColumnName(String column) {
     return AggregationFunctionType.MIN.getName() + "_" + column;
   }
 
+  @Override
+  public String getResultColumnName(String column) {
```

Comment:
Curious if we can have this method in the base class. If the base class calls getFunctionName() that returns the concrete class's getFunctionName(), then we won't need to duplicate this logic in all classes (we would need getFunctionName() though).

Reply:
Yes can be done. Moved the common logic to the default method implementation